### ISSUE TITLE
About FGD score

### ISSUE THREAD

**Issue by Symbolzzz** [andypinxinliu/GestureLSM#4]
Thank you for your contribution to the field! 

I noticed that the paper uses 1000 epochs for training, but in the link you gave, according to the file naming, it seems to be 540 epochs, I re-trained the model locally with 540 iterations, but the FGD scores I got are not the same as yours, can you answer my doubts? Or provide the training log? I will be grateful.

**Comment by andypinxinliu** [comment]
May I know what is the result you have got?

I did not save the training logs. However, you can find it in the updated arxiv version appendix, for which I calculated the FGD for every 20 epochs. Noramlly, the model at best results around 500 - 600. 

If the result you have got is from 0.39 - 0.42 range, I would consider it a reasonable result. In this case, all you need to do is to find out some hyperparamters, like dropout, number of layers of cross-attention (i hard code it to be 3, but sometimes 2 makes better result) before the spatial and temporal attention. For inference, you can look for a good guidance scale, it will be in range of 1.5 - 3.5 from my personal experience. 

If you FGD is outside this range (0.39 - 0.42), the result might be abnormal.

**Comment by Symbolzzz** [comment]
Thank you for your patience.

 03-10 14:54:01 | fid score: 0.44043756921982347
 03-10 14:54:01 | align score: 0.7208726084806503
 03-10 14:54:01 | l1div score: 14.329946754748342
 03-10 14:54:01 | total inference time: 29 s for 945 s motion

Here is my result.  I get an FGD of 0.44, maybe a little out of range.  I haven't adjusted the hyperparameters and used 540 epochs of the model.

**Comment by andypinxinliu** [comment]
Did you retrain the eves as well? And is it holistic or just body regions? These can also matter. I retrained just generator and can achieve similar results. I am not sure why it does not work on your side, unfortunately. 😵😵
